_code-samples/checks/js/get-checks.js

```diff
@@ -1,55 +1,57 @@
 'use strict'
-const xrpl = require('xrpl');
+const xrpl = require('xrpl')
 
-const address = "rEYyvXCwv1WJ6a8hW4sBx5afXbJTK97bbR"; // <-- Change this to your account
-
-const main = async () => {
+async function main() {
   try {
     // Connect ----------------------------------------------------------------
-    const client = new xrpl.Client('wss://s.altnet.rippletest.net:51233');
-    await client.connect();
-
-    let currMarker = null;
-    let account_objects = [];
+    const client = new xrpl.Client('wss://s.altnet.rippletest.net:51233')
+    await client.connect()
 
-    // Loop through all account objects until marker is undefined --------------
+    // Loop through account objects until marker is undefined -----------------
+    let current_marker = null
+    let checks_found = []
```

Comment:
are we purposely keeping the boilerplate code duplicated in each file to provide separate examples? Or could we abstract it for groups of samples since the main difference is the payload object

Reply:
Yeah, the code samples are meant to be something you can actually run with minimal modification. I think including all the necessary boilerplate code helps with context, so readers don't have to guess how much additional setup is needed before the code itself can run. We don't know which code sample they'll find first, after all.

I've been writing the actual tutorials to mostly just showcase the sections that are different, which is hopefully the right balance—users see the unique parts in the tutorial, but they can click through to the full sample code to see how it fits into a fully running sample.

Reply:
makes sense!